docs/content/en/docs/concepts/tracing-policy/hooks.md

```diff
@@ -212,6 +212,64 @@ spec:
 This example shows how to use uprobes to hook into the readline function
 running in all the bash shells.
 
+## LSM BPF
+
+LSM BPF programs allow runtime instrumentation of the LSM hooks by privileged
+users to implement system-wide MAC (Mandatory Access Control) and Audit policies
+using eBPF.
+
+List of LSM hooks which can be instrumented can be found in `security/security.c`.
+
+To verify if BPF LSM is available use the following command:
+
+```
+$ cat /boot/config-$(uname -r) | grep BPF_LSM
+CONFIG_BPF_LSM=y
+```
+
+If the output contains `CONFIG_BPF_LSM=y`, BPF LSM is supported.
```

Comment:
This is a nit only on doc, could you [separate the cmd from its output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output)?

````suggestion
```shell
cat /boot/config-$(uname -r) | grep BPF_LSM
```

The output should be similar to this if BPF LSM is supported:
```
CONFIG_BPF_LSM=y
```
````

Reply:
if you want to merge this as it's green and approved, and fix the doc later it's completely okay

Reply:
No problem, I'll fix it right now!